config/config.go

```diff
@@ -20,7 +20,7 @@ type Config struct {
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	MongoConfig
 	NavigationCacheMaxAge time.Duration `envconfig:"NAVIGATION_CACHE_MAX_AGE"`
-	TopicAPIURL           string        `envconfig:"NAVIGATION_CACHE_MAX_AGE"`
+	TopicAPIURL           string        `envconfig:""`
```

Comment:
Should we be removing this? I can't see it in dp-configs.

Reply:
TopicAPIURL should not be removed!, it is used in many places :- 
api.go , topic_private.go, config/config.go sdk/client.go and service/service.go 
		TopicAPIURL:           "http://localhost:25300",
dp-configs read  "BIND_ADDR